model/flow/transaction.go

```diff
@@ -273,16 +273,18 @@ func (tb *TransactionBody) SignEnvelope(
 	return nil
 }
 
-// SignMessageWithTag signs the message (tag + message) with the specified account key.
+// SignMessageWithTag signs the data (transaction_tag + message) with the specified private key
+// and hasher.
 //
-// This function returns an error if the signature cannot be generated.
+// This function returns an error if:
+//  - crypto.InvalidInputsError if the private key cannot sign with the given hasher
+//  - other error if an unexpected error occurs
 func (tb *TransactionBody) SignMessageWithTag(
```

Comment:
Does it make sense to rename this to `SignTransactionMessage`?

Reply:
How about just `Sign`, so it would be a method `TransactionBody.Sign` ?

Reply:
https://github.com/onflow/flow-go/pull/2171/commits/dcd8a01ce5f9031960837913cf9fccb0a4e0f859